default/python/AI/CombatRatingsAI.py

```diff
@@ -411,3 +411,18 @@ def rating_needed(target, current=0):
         return 0
     else:
         return target + current - 2 * (target * current)**0.5
+
+def rating_difference(first_rating, second_rating):
+
+    """Estimate the rating of an additional force rating needed to be added to the lesser of the given ratings
```

Comment:
It takes me some time and checks other code, to understand what this function does.  Docstring is not helpful.

maybe something like that: 

`Return absolute difference between ratings. Keep in mind that rating is not the linear function.`